src/nimble.nim

```diff
@@ -445,7 +445,7 @@ proc installFromDir(dir: string, requestedVer: VersionRange, options: Options,
   createDir(pkgDestDir)
   # Copy this package's files based on the preferences specified in PkgInfo.
   var filesInstalled = initSet[string]()
-  for file in getInstallFiles(realDir, pkgInfo, options):
+  discard forEachInstallFile(realDir, pkgInfo, options) do(file: string) -> bool:
```

Comment:
Is the return `bool` value necessary? If not please remove it.

Rename to `iterInstallFiles`.

Nit: add space after `do`.